lesson02/assignment/css/_components.content.scss

```diff
@@ -1,17 +1,117 @@
-html {
-    box-sizing: border-box;
+/*style for outter gray box*/
+.main {
+    width: 951px;
+    height: 474px;
+    background-color: $outter-gray; /*variable*/
+    border: 0.5px solid #d3d3d3;
+    border-radius: 5px;
+    
+    /*media query for outter gray box*/
+    @include for-phone-only {
+        width: 100%;
+        height: 100%;
+    }
 }
 
-*,
-*:after,
-*:before {
-    box-sizing: inherit;
+/*style for white box*/
+.background {
+    width: 847px;
+    height: 383px;
+    background-color: white;
+    display: flex;
+    justify-content: space-evenly;
+    margin: 45px;
+    
+    /*media query for white box*/
+    @include for-phone-only {
+        width: 80%;
+        height: 100%;
+        display: block;
+    }
 }
 
-.main {
-    padding: 2rem;
+/*style for photo/name/title*/
+.photo-title {
+    float: left;
+    padding-top: 30px;
+    align-self: center;
+
+    /*media query for photo/name/title block*/
+    @include for-phone-only {
+        width: 75%;
+        float: none;
+    }
+    
+    /*First instance of nesting*/
+    &__headshot {
+      width: 225px;
+      height: 222px;
+      border-radius: 50%;
+
+      /*media queries for headshot*/
+      @include for-phone-only {
+          margin-left: 20%;
+      }
+
+      @include for-smaller-phone {
+          margin-left: 10%;
+      }
+    }
+
+    &__name {
+      font-size: 26px;
+      font-weight: bold;
+      color: #707070;
+      font-family: $font; /*variable*/
+      text-align: center;
+    }
+
+    &__title {
+      color: rgb(112, 112, 112);
+      font-size: 20px;
```

Comment:
Font size should also always use `rem` or `%` as a unit of measurement. Setting it to a fixed pixel height means that users who need to zoom to a larger font size will not be able to do so. You don't want to limit your users in that way.